experiments/ClimaEarth/run_dry_held_suarez.jl

```diff
@@ -13,15 +17,11 @@ import YAML
 ## ClimaESM packages
 import ClimaAtmos as CA
 using ClimaCore
```

Comment:
could change ClimaCore from `using` to `import` too

Reply:
Good catch!